### ISSUE TITLE
Replace literal constants with parameters

### ISSUE THREAD

**Issue by gold2718** [NorESMhub/OSLO_AERO#34]
There are many literal constants in Oslo Aero that should be documented.
Some of them should be replaced with shared parameters such as the constant, `287.04`, in src/aero_model.F90 (should be `rair` from physconst) while others should be turned into module or subroutine parameters with appropriate names and source documentation.

Note that changing `287.04` to `rair` will change answers so this should be done in a separate commit.

**Comment by gold2718** [comment]
Also, all constants (parameters or literal) should be explicitly type `_r8`. One such file that needs fixing is src/oslo_areo_ocean.F90 .

**Comment by oyvindseland** [comment]
Yes I agree and given the development phase it is the correct time to do so.

Do you want help with going through the variables?
I think we should do the cleaning up in 2.5. Or should we also do it for older versions? I presume given the rewriting from 2.1 to 2.3 we can do the work in 2.3 and then merge.

**Comment by gold2718** [comment]
I think it depends on how the scheduling falls out. Since some of this work might change answers, it should be in a separate commit (so we can verify that nothing bad has happened and we are just dealing with tiny changes). I would say it is a lower priority than getting 2.3 released so I would like to finish the other 2.3 work first.
